saleor/graphql/product/mutations/products.py

```diff
@@ -98,6 +98,10 @@ class Meta:
     @classmethod
     def clean_input(cls, info, instance, data):
         cleaned_input = super().clean_input(info, instance, data)
+        description = cleaned_input.get("description")
+        cleaned_input["description_plaintext"] = (
+            clean_editor_js(description, to_string=True) if description else ""
+        )
```

Comment:
We should add 2 tests for that. 
1. Test for creating category with description.
2. Test for updating category with description.